pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -2853,6 +3017,289 @@ void FullscreenUI::DrawPathSetting(SettingsInterface* bsi, const char* title, co
 	}
 }
 
+void FullscreenUI::DrawTextSetting(SettingsInterface* bsi, const char* title, const char* summary, const char* section, const char* key,
+	const char* default_value, bool enabled /* = true */, float height /* = ImGuiFullscreen::LAYOUT_MENU_BUTTON_HEIGHT */,
+	ImFont* font /* = g_large_font */, ImFont* summary_font /* = g_medium_font */)
+{
+	const bool game_settings = IsEditingGameSettings(bsi);
+	const std::optional<SmallString> value(
+		bsi->GetOptionalSmallStringValue(section, key, game_settings ? std::nullopt : std::optional<const char*>(default_value)));
+
+	if (MenuButton(title, summary, value.has_value() ? value->c_str() : FSUI_CSTR("Use Global Setting")))
+	{
+		const char* prompt = FSUI_CSTR("Enter a new value:");
+
+		// For network settings, show format hints
+		if (std::string(section) == "DEV9/Eth")
+		{
+			if (std::string(key) == "PS2IP")
+				prompt = FSUI_CSTR("Enter PS2 IP address (format: xxx.xxx.xxx.xxx):");
+			else if (std::string(key) == "Mask")
+				prompt = FSUI_CSTR("Enter subnet mask (format: xxx.xxx.xxx.xxx):");
+			else if (std::string(key) == "Gateway")
+				prompt = FSUI_CSTR("Enter gateway address (format: xxx.xxx.xxx.xxx):");
+			else if (std::string(key) == "DNS1")
+				prompt = FSUI_CSTR("Enter primary DNS address (format: xxx.xxx.xxx.xxx):");
+			else if (std::string(key) == "DNS2")
+				prompt = FSUI_CSTR("Enter secondary DNS address (format: xxx.xxx.xxx.xxx):");
+		}
```

Comment:
This doesn't seem needed now you have `DrawIPAddressSetting()`

Reply:
Removed